src/NuGetForUnity/Editor/NugetConfigFile.cs

```diff
@@ -74,6 +74,37 @@ public class NugetConfigFile
         /// </summary>
         public bool ReadOnlyPackageFiles { get; set; }
 
+        public string PackagesConfigPath;
+
+        public string RelativePackagesConfigPath
+        {
+            get => NugetHelper.GetProjectRelativePath(PackagesConfigPath);
+
+            private set
+            {
+                var tempValue = Environment.ExpandEnvironmentVariables(value);
```

Comment:
If we support EnvironmentVariables inside the path. We need to store the "non expanded value" as loading and saving the NuGet.config file shouldn't change the value.

Reply:
Should we support env variables for this? Seems weird that you depend on different packages in different environments considering how packages are usually made.

Reply:
Normally the `package.config` should live in the same GIT as the source code therefore relative path's should be enough to handle all scenarios.